python/paddle/nn/functional/loss.py

```diff
@@ -2229,3 +2230,124 @@ def hinge_embedding_loss(input, label, margin=1.0, reduction='mean', name=None):
         return paddle.sum(loss, name=name)
     elif reduction == 'none':
         return loss
+
+
+def triplet_margin_loss(input,
+                        positive,
+                        negative,
+                        swap=False,
+                        p=2,
+                        eps=1e-6,
+                        margin=1.0,
+                        reduction='mean',
```

Comment:
some common parameters with `paddle.nn.TripletMarginLoss` are better to have same order ? e.g. margin, p, epsilon, swap, reduction